.gitignore

```diff
@@ -51,3 +51,4 @@ environment.y*
 coverage/
 coverage.info
 .coverage
+yaml/
```

Comment:
I think it would be more consistent with our other build steps for this to get generated as a subdirectory of `build`, in which case it will automatically be excluded.

Reply:
I'll hold off on this then, as this comes from the `YamlSourceGenerator` ... this should follow the same approach as `CLibSourceGenerator`, which currently drops things into the root folder. A resolution is, at least implicitly, part of the roadmap in Cantera/enhancements#220. I'll add this on the todo list there to keep track of this.